src/app/components/Main/Promo/Featuring/index.tsx

```diff
@@ -1,25 +1,40 @@
 import React from "react";
 import FeaturingItem from "./FeaturingItem";
+import snacks_img from "./../../../../assets/snacks_image.jpeg";
```

Comment:
```suggestion
import snacks_img from "@/assets/snacks_image.jpeg";
```
You can use relative import using `@`  this is due to `tsconfig.json` path configuration

![image](https://github.com/PixelGore/beer-bear/assets/57370488/c2d60953-9bb1-49de-8125-bc1032d5afe3)